pkg/controller/ttlafterfinished/ttlafterfinished_controller.go

```diff
@@ -239,16 +239,17 @@ func (tc *Controller) processJob(key string) error {
 		Preconditions:     &metav1.Preconditions{UID: &fresh.UID},
 	}
 	klog.V(4).Infof("Cleaning up Job %s/%s", namespace, name)
+	startDeletion := tc.clock.Now()
 	if err := tc.client.BatchV1().Jobs(fresh.Namespace).Delete(context.TODO(), fresh.Name, options); err != nil {
 		return err
 	}
-	metrics.TimeToDeletionSeconds.Observe(timeToDeletion.Seconds())
+	metrics.TimeToDeletionSeconds.Observe(timeToProcess.Seconds() + time.Since(startDeletion).Seconds())
```

Comment:
I would prefer to just do: `time.Since(expireTime(job))`

Reply:
yeah, me too, I got distracted by all the usage of UTC, so I didn't want to touch that code. But reading a bit about the semantics of that, it turns out this is all not necessary. Since it can be confusing when reading the code, I cleaned that up as well.